/**:
  ros__parameters:
    target_cloud_path: "/home/hakan/projects/final_ws/src/pcl_regisration/data/capture0001.pcd"
    input_cloud_path: "/home/hakan/projects/final_ws/src/pcl_regisration/data/capture0002.pcd" 
    output_cloud_path: "/home/hakan/projects/final_ws/src/pcl_regisration/data/transformed.pcd"